requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy>=1.17
+numpy
```

Comment:
I am sorry. This was just fine. I thought that #74 was already merged. So this change is not required and the pinning that is introduced over there is getting unnecessary.

Reply:
makes sense!